modules/slugs/includes/qtranslate-slug-import.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * Legacy meta and options from QTS plugin.
+ */
+const QTX_SLUGS_LEGACY_QTS_META_PREFIX    = '_qts_slug_';
+const QTX_SLUGS_LEGACY_QTS_OPTIONS_PREFIX = '_qts_';
+const QTX_SLUGS_LEGACY_QTS_OPTIONS_NAME   = 'qts_options';
+
+/**
+ * Check if slugs meta should be imported from the legacy QTS postmeta and termmeta.
+ *
+ * @return string messages giving details, empty if new meta found or no legacy meta found.
+ */
+function qtranxf_slugs_check_import_qts() {
+    global $wpdb;
+
+    /**
+     * Generic function that counts the slugs meta, legacy (QTS) or new (QTX).
+     *
+     * @param string $table name of the meta table (postmeta, termmeta)
+     * @param string $prefix prefix for the meta key
+     * @param string[] $msg array of messages, updated
+     *
+     * @return void
+     */
+    $count_slugs = function ( $table, $prefix, &$msg ) use ( $wpdb ) {
+        $results = $wpdb->get_var( "SELECT count(*) FROM  $table WHERE meta_key like '$prefix%'" );
+        if ( $results ) {
+            $msg[] = sprintf( __( "Found %s slugs from $table.", 'qtranslate' ), $results );
+        }
+    };
+
+    $msg = [];
+    $count_slugs( $wpdb->postmeta, QTX_SLUGS_META_PREFIX, $msg );
+    $count_slugs( $wpdb->termmeta, QTX_SLUGS_META_PREFIX, $msg );
+    if ( ! empty( $msg ) ) {
+        // Found some post/term meta with the new keys, no import to suggest (it can still be done manually).
+        return '';
+    }
+
+    $msg = [];
+    $count_slugs( $wpdb->postmeta, QTX_SLUGS_LEGACY_QTS_META_PREFIX, $msg );
+    $count_slugs( $wpdb->termmeta, QTX_SLUGS_LEGACY_QTS_META_PREFIX, $msg );
+
+    return empty ( $msg ) ? $msg : implode( '<br>', $msg );
+}
+
+/**
+ * Import slugs meta by duplicating the legacy QTS postmeta and termmeta for QTX.
+ *
+ * @param bool $db_commit true to commit changes, false for dry-run mode.
+ *
+ * @return string messages giving details.
+ */
+function qtranxf_slugs_import_qts_meta( $db_commit ) {
+    global $wpdb;
+
+    $new_prefix = QTX_SLUGS_META_PREFIX;
```

Comment:
If options import is requested while slugs module is inactive `QTX_SLUGS_META_PREFIX` is undefined, resulting in fatal error .
Checkbox can be shown conditionally, `QTX_SLUGS_META_PREFIX` defined outside the module  or function can return with notice if test is negative.

Reply:
covered by 5ed9b2e7088b0ca2f408a5e49bf366bfd7d36baf